emails/2023-02-10.md

```diff
@@ -0,0 +1,25 @@
+This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub. 
+
+This week, we had [an iconic video](https://www.youtube.com/watch?v=jb2AvF8XzII&t=128s) feature us as one of the 15 future databases. Thanks [Fireship](https://www.youtube.com/@Fireship)! For this week's joke, I'm just going to ask [you watch the first 5 seconds of the video](https://www.youtube.com/watch?v=jb2AvF8XzII&t=128s). Fireship is funnier and more daring than I could ever be in this email.
+
+### Dolt for Backups
+
+Astute reader will notice a lot more content focused on how and why to use Dolt. We've done a good job building a great product people are interested in.But we haven't done a good enough job explaining how and why to use it. We're making a concerted effort to change that this year, starting with written blogs, because that's what we're best at.
+
+In service of the above theme, we have [Dolt for Backups](). This could have been called Dolt your dumps but that was a way worse title. This article explains some of the features you get using a detailed example if you store a copy of your MySQL dumps in Dolt. Like [the binlog replication](https://www.dolthub.com/blog/2022-12-16-coming_soon_binlog_replication/) I've been touting, this is purely additive, you cans still run MySQL. Get time travel, and audit log of every cell, and the ability to clone for development just by sticking a copy of your dumps in Dolt.
```

Comment:
```suggestion
In service of the above mission, we have [Dolt for Backups](https://www.dolthub.com/blog/2023-02-06-dolt-backups/). This could have been called Dolt your dumps but that was a way worse title. This article explains some of the features you get, using a detailed example, if you store a copy of your MySQL dumps in Dolt. Like [the binlog replication](https://www.dolthub.com/blog/2022-12-16-coming_soon_binlog_replication/) I've been touting, this is purely additive, you can still run MySQL. Get time travel, an audit log of every cell, and the ability to clone for development just by sticking a copy of your dumps in Dolt.
```